ecorest/config/main.go

```diff
@@ -2,19 +2,49 @@ package config
 
 import (
 	"github.com/OpenTreeMap/otm-ecoservice/eco"
+	"os"
 )
 
+type server struct {
+	Host string
+	Port string
+}
+
+type data struct {
+	Path string
+}
+
 type Config struct {
 	Database eco.DBInfo
-	Data     struct {
-		Path string
-	}
-	Server struct {
-		Host string
-		Port string
+	Data     data
```

Comment:
Yeah, that simplifies things a bit.  Done in https://github.com/OpenTreeMap/otm-ecoservice/commit/ae4773fd4770d5908da22e997a2f49ec4e33e8d5